site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
+title: Configure Application Email Templates
+description: I want to use application specific email templates to override Tenant level templates
+navcategory: developer
+---
+
+:page-liquid:
+
+:sectnumlevels: 0
+
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+== Application Specific Email Templates
+
+Before you begin, complete the following guides:
+
+    * link:docs/v1/tech/getting-started/_5-minute-setup-guide.adoc[5 minute setup guide]
+    * link:docs/v1/tech/email-templates/configure-email[Configure Email]
+    * link:/docs/v1/tech/core-concepts/applications[Application overview]
+    * link:docs/v1/tech/tutorials/register-user-login-api[Register a user and login]
+
+In FusionAuth, a Tenant can be configured to send transactional emails using default Email Templates that ship with FusionAuth, such as `Email Update`, `Forgot Password`, `Suspicious login`, etc. See the full list of email templates that can be configured here: link:docs/v1/tech/core-concepts/tenants#template-settings[Template Settings]
+
+To begin, we'll configure our email template at the Tenant level to ensure they are sent when a user starts the link:docs/v1/tech/apis/users[Start Forgot Password Workflow].
+
+Next, navigate to [breadcrumb]#Tenant -> Email -> Template Settings -> Forgot Password#. Select `Forgot Password` from the dropdown, then click the blue save icon. When a user clicks the `Forgot Password` link on the hosted login page or calls the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[forgot password API], this template will be used to email the user to start the workflow to change their password for all applications in the specified Tenant - so long as they don't have a template set at the Application level.
```

Comment:
So, if you are going against a new FusionAuth install, you won't see any templates here. You have to add them first, under customizations -> email templates. Can you please include that step, maybe copying from the email templates forgot password example?

![image](https://user-images.githubusercontent.com/91825/214630052-7ec73368-4425-4b2f-a48a-643711ed0142.png)

Reply:
This is odd. I just pulled in a fresh docker image and spun up a container and I had some default email templates at the Tenant level. I'll include a code snippet of the Forgot Password email template and directions for how to create it, just to be sure. @mooreds